autonomy/cli/scaffold_fsm_templates/components.py

```diff
@@ -276,6 +279,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
     Params = BaseParams
     Requests = BaseRequests
+    BenchmarkTool = BaseBenchmarkTool
```

Comment:
It also needs to be added in the skill config template

Reply:
According to @angrybayblade , since the skill.yaml scaffolding happens on open-aea side, what we will need to do is, after the FSM scaffolding is done:
- Load the skill config 
- Add the benchmark tool model definition
- Save it
I need to look into it, but as I'm still finding other small template errors, I think it's better to leave this PR open for now and I focus on the contribution service adding here the errors I find along the way. Once finished with it, then I look into this issue and fix it here.

Reply:
We already update the skill yaml: https://github.com/valory-xyz/open-autonomy/blob/main/autonomy/cli/scaffold_fsm.py#L351

Reply:
@dvilelaf please address this - it's possible!

Reply:
Done